student.rb

```diff
@@ -3,11 +3,16 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom)
-    super(age, name: 'Unknown', parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] I've noticed you are using a variable called `classroom` although it has not been defined anywhere. you should add the `classroom` variable as one of the parameters
- [ ] i notice you are calling `super` however, the last two arguments to `super` should be named variables otherwise this code won't run.

```suggestion
  def initialize(age, name: 'Unknown', parent_permission: true)
    super(age, name: name, parent_permission: parent_permission)
    @classroom = nil
  end
```